extensions/amp-viewer-integration/0.1/messaging/messaging.js

```diff
@@ -58,9 +55,10 @@ export function parseMessage(message) {
   if (message.charAt(0) != '{') {
     return null;
   }
-  return /** @type {?Message} */ (tryParseJson(message) || null);
+  return /** @type {?Message} */ (JSON.parse(/** @type {string} */ (message)));
```

Comment:
This is a big change - the previous version failed silently. You should probably preserve that. Anyone can send a message and it's not guaranteed to be what we expect. So it's best to shield ourselves and overspam error logs.